birdhouse/scripts/logging.include.sh

```diff
@@ -58,7 +58,14 @@ export LOG_CRITICAL="${REG_BG_BOLD}CRITICAL${NORMAL}: "  # to report misuse of f
 log_dest() {
     level=$1
     option=$2
-    read log_line || true
+    log_line=
+    while IFS= read -r line; do
+        if [ -z "${log_line}" ]; then
+            log_line="$line"
+        else
+            log_line="${log_line}\n          ${line}"
```

Comment:
I'm guessing these spaces are based on the length of `LOG_DEBUG`, `LOG_INFO`, etc. so the message is left-aligned after the level? If so, probably better if we compute the length automatically to have the trace of where they come from and make sure they stay aligned. Or, simply a comment otherwise.